src/main/java/com/band/api/controller/HelloWorldController.java

```diff
@@ -16,8 +16,9 @@ public HelloWorldController(HelloWorldService helloWorldService) {
         this.helloWorldService = helloWorldService;
     }
 
-    @GetMapping("/")
-    public HelloWorldDto helloWorld() {
-        return helloWorldService.createHelloDto();
-    }
+//    @GetMapping("/")
```

Comment:
Don't leave commented code (remove this method or the whole class since we don't need it).